internal/core/operations/switch_active_version/switch_active_version_executor.go

```diff
@@ -155,11 +160,21 @@ func (ex *SwitchActiveVersionExecutor) deleteNewCreatedRooms(ctx context.Context
 	return nil
 }
 
-func (ex *SwitchActiveVersionExecutor) startReplaceRoomsLoop(ctx context.Context, logger *zap.Logger, maxSurgeNum int, scheduler entities.Scheduler) error {
+func (ex *SwitchActiveVersionExecutor) startReplaceRoomsLoop(ctx context.Context, logger *zap.Logger, maxSurgeNum int, scheduler entities.Scheduler, op *operation.Operation) error {
 	logger.Info("replacing rooms loop - start")
 	roomsChan := make(chan *game_room.GameRoom)
 	errs, ctx := errgroup.WithContext(ctx)
 
+	var totalRoomsAmount int
+	var err error
+	err = retry.Do(func() error {
```

Comment:
This code will retry just one more time?

Reply:
Nope! This method is using default options from the library, which is 10 retries (with increasing delay)